--- DeepStream-Yolo-master/nvdsinfer_custom_impl_Yolo/nvdsparsebbox_Yolo.cpp
+++ DeepStream-Yolo-master/nvdsinfer_custom_impl_Yolo/nvdsparsebbox_Yolo.cpp
@@ -34,7 +34,7 @@
 static NvDsInferParseObjectInfo
 convertBBox(const float& bx1, const float& by1, const float& bx2, const float& by2, const uint& netW, const uint& netH)
 {
-  NvDsInferParseObjectInfo b;
+  NvDsInferParseObjectInfo b{};
 
   float x1 = bx1;
   float y1 = by1;

--- DeepStream-Yolo-master/nvdsinfer_custom_impl_Yolo/nvdsparsebbox_Yolo_cuda.cu
+++ DeepStream-Yolo-master/nvdsinfer_custom_impl_Yolo/nvdsparsebbox_Yolo_cuda.cu
@@ -46,6 +46,7 @@
 
   if (maxProb < preclusterThreshold[maxIndex]) {
     binfo[x_id].detectionConfidence = 0.0;
+    binfo[x_id].rotation_angle = 0.0;
     return;
   }
 
@@ -65,6 +66,7 @@
   binfo[x_id].height = fminf(float(netH), fmaxf(float(0.0), by2 - by1));
   binfo[x_id].detectionConfidence = maxProb;
   binfo[x_id].classId = maxIndex;
+  binfo[x_id].rotation_angle = 0.0;
 }
 
 static bool NvDsInferParseCustomYoloCuda(std::vector<NvDsInferLayerInfo> const& outputLayersInfo,
